README.md

```diff
@@ -39,18 +39,28 @@ Set up project with Docker
     docker exec -it linkedevents-backend python manage.py event_import helmet --events
     # Import events from Espoo
     docker exec -it linkedevents-backend python manage.py event_import espoo --events
+    # Import organizations
```

Comment:
Perhaps we should add `(optionally)` to all these phases 5-8, and explain that the empty database works already without running any Helsinki-specific stuff!